ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -2274,6 +2274,28 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-nutanix-ipi
+- as: nutanix-ipi-zones-f14
+  cluster: build01
+  cron: 35 15 14,28 * *
+  steps:
+    cluster_profile: nutanix-qe-zone
+    env:
+      EXTRACT_MANIFEST_INCLUDED: "true"
```

Comment:
Can you help also update other nutanix jobs to add this ENV for `nutanix-ipi-tp` and `nutanix-ipi-proxy-fips-f28-ui`.

Reply:
What about I open another PR to do it? I'd hope this PR get merged days before, it's a little late.

Reply:
ok